src/base/bittorrent/torrent.h

```diff
@@ -129,6 +129,7 @@ namespace BitTorrent
         static const qreal MAX_RATIO;
         static const int MAX_SEEDING_TIME;
         static const int MAX_INACTIVE_SEEDING_TIME;
+        static const qreal INTEREST_RATE_PERIOD;
```

Comment:
I don't believe that it should be declared public. Just move its declaration in "torrentimpl.cpp".

Reply:
Thanks! `INTEREST_RATE_PERIOD` has been renamed to `MONTH_SECONDS` and moved to `torrentimpl.cpp`.